libs/elements/layout/images/src/lib/components/elewa-image-container/elewa-image-container.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import { _images as mockImages } from '@elewa-website/data/ui/images'
+import { ImageConfig } from '@elewa-website/models/schema/ui/images';
+
+@Component({
+  selector: 'elewa-website-elewa-image-container',
+  templateUrl: './elewa-image-container.component.html',
+  styleUrls: ['./elewa-image-container.component.scss'],
+})
+export class ElewaImageContainerComponent implements OnInit {
+  @Input() images!: ImageConfig[]
+
+  ngOnInit(){
+    this.images = mockImages
+  }
```

Comment:
Since this is a reusable component we only expect one item. No need to initialise this as an array of imageConfigs and also no need to submit the initialisation of this.images with the mockImages, that's for your own testing.